src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfUserNameService.cs

```diff
@@ -20,7 +20,7 @@ public override void Validate(string userName, string password)
                 throw new ArgumentNullException();
             }
 
-            if (!(userName == "test1" && password == "test1pwd") && !(userName == "test2" && password == "test2pwd"))
+            if (!(userName == "test1" && password == "Mytestpwd1") && !(userName == "test2" && password == "test2pwd"))
```

Comment:
We currently do not share code between client and server. We need a design discussion if we want to change it.
One thing we need to consider is how to keep IIS compile code for us.